client/common/sass/components/_buttons.sass

```diff
@@ -28,7 +28,8 @@
 			content: url(../icons/arrow-left.svg)
 			height: 24px
 			width: 16px
-			border: 3px solid
+			// 2.95px fixes border overlap issue on iOS
+			border: 2.95px solid
```

Comment:
With 2.95px, it looks overlapping in firefox and chrome. I think maybe we can keep 3px for now, and find a better way to solve this later? @harrislapiroff